### PR TITLE
app-emulation/docker-compose: ARM64 support to ebuild.

### PR DESCRIPTION
**Pull request by samip5**
For some unknown reason it did not have the required tag so now it does.

Please let me know if this is not the right way to do it.

Related https://github.com/docker/compose/issues/7370

### CODE REVIEW

**Comment by hartwork** [comment]
PS: Running `repoman full` may help finding dependencies that also lack this keyword.

**Comment by samip5** [comment]
> Hi!
> 
> > Please let me know if this is not the right way to do it.
> 
> Please don't do a revbump for adding a keyword. Just edit existing `docker-compose-1.26.0.ebuild` in-place. Thank you!

Okey. I did edit the existing one, and also installing repoman atm.

**Comment by thesamesam** [comment]
The preferred way is to file a keywording request bug on Bugzilla for ~arm64. Thanks.

File: dev-python/cached-property/cached-property-1.5.1.ebuild [hunk 1]
```diff
@@ -12,7 +12,7 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 ppc64 x86"
+KEYWORDS="amd64 arm64 ppc64 x86"
```
Comment (hartwork) [inline dev-python/cached-property/cached-property-1.5.1.ebuild:15]: Don't add straight to stable, please. Please use `~am64`, initially.

Comment (samip5) [reply dev-python/cached-property/cached-property-1.5.1.ebuild:15]: Ah, alright.

**Comment by thesamesam** [comment]
Like I said, this isn't the way that the arm64 team works -- I will happily process your keywording bug, but we like to thoroughly test everything first, so there is no need for the PR.

**Comment by samip5** [comment]
> Like I said, this isn't the way that the arm64 team works -- I will happily process your keywording bug, but we like to thoroughly test everything first, so there is no need for the PR.
> 
> We then commit ourselves when ready.

I get that, but I want to help but it seems that the arm64 keyword was missing for a larger reason than I thought originially.

**Comment by thesamesam** [comment]
That's okay -- we then expand the bug to include all needed dependencies. Often, a lot of packages never had a reason to be keyworded until a package like this needs it.

**Comment by hartwork** [comment]
@samip5 sorry for not sending you to bugzilla right away. I guess @thesamesam is right. I don't remember doing keywording outside of bugzilla either when I think about it.

**Comment by samip5** [comment]
Okay, well lets see what the QA checks now say with the current commits. I think there were at least one dependency that does not have correct tags etc.

Is the keywording usually done inside bugzilla? How does that actually work then?

**Comment by thesamesam** [comment]
@samip @hartwork No worries.

The general procedure is:
- File a bug on Bugzilla in the Gentoo Linux > Keywording component

- Describe if the package works for you, or if you had any problems with some functionality

- Ideally try out tests (FEATURES=test emerge -av ....) but this is not needed. If there's any problems, file them as a new bug, and block this bug by them.

- Add e.g. 'app-emulation/docker-compose ~arm64' to the package list.

- Add CC-ARCHES keyword because we don't generally need maintainer ACKs to add a keyword, they can just un-CC if they disagree.

- Wait for Nattka to give a sanity check (+) or (-).

- If it is - (it would have been here, because of all the missing deps like you saw), you, the maintainer, or the arch teams may modify the package list to make it + based on the Nattka comment (akin to the CI output).

- Arch teams will then get to the bug and commit the keywording themselves if it passes their tests. If not, they will make blocker bugs.

The reason we don't do keywording/stables by PR is because we like to run quite exhaustive tests, especially in ~arm64 because we have the hardware for it. We sometimes find some useful errors, not usually arch-specific, but worth fixing.

**Comment by thesamesam** [comment]
@samip5 One more thing:

Please make as many keywording (or stable requests -- but don't CC-ARCHES there) as you like, and I will get to them as soon as I can. We're trying to expand the compatibility for arm64 so it's all really appreciated!

You may also want to come chat with us on #gentoo-arm if you aren't already.

**Pull request closed without merging**